my-app/src/app/core/pipes/order-by.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Course } from 'src/app/models/course';
+
+@Pipe({
+  name: 'orderBy'
+})
+export class OrderByPipe implements PipeTransform {
+
+  public transform(courses: Course[]): unknown {
```

Comment:
transform method returns also array of courses

Reply:
done